src/Microsoft.Diagnostics.EventFlow.Consumers.ASPNET/Microsoft.Diagnostics.EventFlow.Consumers.ASPNET.csproj

```diff
@@ -20,16 +20,16 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.0.3" />
-    <PackageReference Include="Microsoft.AspNetCore.Server.IISIntegration" Version="1.0.2" />
-    <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel" Version="1.0.3" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="1.1.1" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.FileExtensions" Version="1.1.1" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="1.1.1" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="1.1.1" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="1.1.1" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.1" />
-    <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="1.1.1" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.0.6" />
```

Comment:
1.* is not good--we do not want anything greater that 1.0.* because 1.1 had breaking changes compared to 1.0

1.0.* is also not good because that means "lowest applicable version from 1.0 series" https://docs.microsoft.com/en-us/nuget/consume-packages/dependency-resolution https://docs.microsoft.com/en-us/nuget/reference/package-versioning That would select 1.0.0 by default, which has known security issues

We could consider something like [1.0.6, 1.1.0). I will do this next time I need to deal with security fixes ☺️

Reply:
By reading the spec, I think 1.0.* will take the highest version:
[Floating versions](https://docs.microsoft.com/en-us/nuget/consume-packages/dependency-resolution#floating-wildcard-versions)

Reply:
The **lowest applicable version** rule restores the lowest possible version of a package as defined by its dependencies. It also applies to dependencies on the application or the class library **unless declared as floating**.

Reply:
Yes, you are right. It is confusing <sigh> Wildcard gets the highest version, the exact number gets your the lowest version. Thanks for clarifying this for me!

Reply:
👍 Well, this helps me understand that better as well.